hack/ginkgo-e2e.sh

```diff
@@ -130,13 +128,12 @@ else
     "--auth_config=${AUTH_CONFIG:-}"
     "--cert_dir=${CERT_DIR:-}"
   )
-  host="https://${KUBE_MASTER_IP-}"
 fi
 
 # Use the kubectl binary from the same directory as the e2e binary.
 export PATH=$(dirname "${e2e}"):"${PATH}"
 "${e2e}" "${auth_config[@]:+${auth_config[@]}}" \
-  --host="$host" \
+  --host="https://${KUBE_MASTER_IP-}" \
```

Comment:
Indeed, it becomes inconsistent but, when using a `.kubeconfig` file, the host to connect to is found in the `.kubeconfig` file and the `--host` parameter is ignored.

The code is there:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/test/e2e/util.go#L120
* If we provide a `.kubeconfig` everything comes from `testContext.kubeConfig`.
* If we provide a `--auth_config`, then we use `testContext.host`.

The goal of my change was to remove the libvirt-coreos specific setting of `--host` because it is not used anymore.

In order to remove that misleading trick, we could move the `--host` parameter to the provider specific `auth_config` section, except for the ones using `kubeconfig`?